src/consumer.rs

```diff
@@ -118,19 +119,20 @@ impl PartitionConsumerJS {
         }
     }
 
-    async fn stream_inner<F: Fn(String, RecordSetWrapper)>(
+    async fn stream_inner<F: Fn(String, String)>(
```

Comment:
this should return `ArrayBuffer` but this is not show stopper. We should add as issue for next release

Reply:
Why? stream_inner returns `Result<(), FluvioError>`, and the callback is used to return the value; `stream_inner` never returns a value direct, it just awaits for incoming messages.